cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -106,19 +107,23 @@ private static int _main() throws UnknownHostException {
             matcher.group(2)
         );
 
+        if (seedCount > executorCount || seedCount <= 0 || executorCount <= 0)
+            throw new IllegalArgumentException("number of nodes (" + executorCount + ") and/or number of seeds (" + seedCount + ") invalid");
+
         final PersistedCassandraFrameworkConfiguration configuration = new PersistedCassandraFrameworkConfiguration(
             state,
             frameworkName,
             cassandraVersion,
             executorCount,
+            seedCount,
             resourceCpuCores,
             resourceMemoryMegabytes,
             resourceDiskMegabytes,
-            healthCheckIntervalSec
+            healthCheckIntervalSec,
+            bootstrapGraceTimeSec
         );
-        final ExecutorCounter executorCounter = new ExecutorCounter(state, 0L);
-        final PersistedCassandraClusterState persistedCassandraClusterState = new PersistedCassandraClusterState(state);
-        final PersistedCassandraClusterHealthCheckHistory persistedCassandraClusterHealthCheckHistory = new PersistedCassandraClusterHealthCheckHistory(state);
+        configuration.numberOfNodes(executorCount);
```

Comment:
these values shouldn't be overwritten here.  If we need to allow for the values to change we will need to create a rest endpoint that can perform the operation and log that it happened.